cmd/runtime_options_test.go

```diff
@@ -113,11 +118,12 @@ func TestRuntimeOptions(t *testing.T) {
 	runtimeOptionsTestCases := map[string]runtimeOptionsTestCase{
 		"empty env": {
 			useSysEnv: true,
+			systemEnv: map[string]string{},
 			// everything else is empty
 			expRTOpts: lib.RuntimeOptions{
 				IncludeSystemEnvVars: null.NewBool(true, false),
 				CompatibilityMode:    defaultCompatMode,
-				Env:                  nil,
+				Env:                  map[string]string{},
```

Comment:
I hope that this change is acceptable. I can't imagine that sys vars can be nil, and is it essential (having `nil` there)? :thinking: